### PR TITLE
Prepared health check with logging for inclusion in C.G.C. documentation

### PR DESCRIPTION
**Pull request by sahanacp-hub**
Intending to use this snippet on the following page:
https://cloud.google.com/load-balancing/docs/health-check-logging

```release-note:none
```

### CODE REVIEW

File: health-check-tcp/main.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# [START cloudloadbalancing_health_check_tcp_with_logging]
+resource "google_compute_health_check" "health-check-tcp-with-logging" {
```
Comment (betsy-lichtenberg) [inline health-check-tcp/main.tf:2]: Please use underscores instead of dashes in resource names

File: health-check-tcp/test.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
```
Comment (betsy-lichtenberg) [inline health-check-tcp/test.yaml:1]: Please delete this file. We should only skip testing when necessary.

File: health-check-tcp/main.tf (hunk 1 above)
Comment (sahanacp-hub) [reply health-check-tcp/main.tf]: Replaced dashes with underscores

File: health-check-tcp/test.yaml (hunk 2 above)
Comment (sahanacp-hub) [reply health-check-tcp/test.yaml]: Done!

**Review by betsy-lichtenberg — APPROVED** [review]

**Pull request merged**